scipy/interpolate/meson.build

```diff
@@ -123,25 +128,33 @@ py3.extension_module('_rgi_cython',
 
 __fitpack_lib = static_library('__fitpack',
     ['src/__fitpack.h', 'src/__fitpack.cc'],
-    dependencies:[lapack, np_dep, py3_dep],
+    dependencies:[lapack_ilp64, np_dep, py3_dep],
+    c_args: c_flags_ilp64
 )
 
 __fitpack_dep = declare_dependency(
     link_with: __fitpack_lib,
 )
 
+
 py3.extension_module('_dierckx',
     ['src/_dierckxmodule.cc'],
     include_directories: 'src/',
     dependencies: [np_dep, __fitpack_dep],
     link_args: version_link_args,
+    cpp_args: c_flags_ilp64,
     install: true,
     subdir: 'scipy/interpolate'
 )
 
-# TODO: Add flags for 64 bit ints
-py3.extension_module('_fitpack',
+# Build _fitpack and dfitpack extensions: both are FITPACK wrappers.
+# XXX: some functions from dfitpack use the F_INT macro defined in dfitpack.pyf, while
+#      others rely on the build flags only. Consider cleaning this up at some point.
+
+
+_fitpack = py3.extension_module('_fitpack',
```

Comment:
Defining `_fitpack` here doesn't seem to do anything. Please consider reverting to what it was before:
```meson
py3.extension_module('_fitpack',
```